api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -33,6 +33,9 @@ type KubeadmConfigSpec struct {
 	// JoinConfiguration is the kubeadm configuration for the join command
 	// +optional
 	JoinConfiguration *kubeadmv1beta1.JoinConfiguration `json:"joinConfiguration,omitempty"`
+	// AdditionalUserDataFiles specifies extra files to be passed to user_data upon creation.
+	// +optional
+	AdditionalUserDataFiles []Files `json:"additionalUserDataFiles,omitempty"`
```

Comment:
Does this need to be wired up anywhere in the cloudinit code?

Reply:
Yes but I'm going to file another issue for that.